apis/facebook/api/src/facebook_api.e

```diff
@@ -124,9 +127,53 @@ feature -- Facebook: Get User
 			end
 		end
 
+	user_likes (a_path: STRING; a_params: detachable FB_PAGE_PARAMETER): detachable STRING
+		do
+			if
+				a_params /= Void and then
+				attached a_params.parameters as l_parameters
+			then
+				api_get_call (facebook_url (a_path, l_parameters ), Void)
+			else
+				if a_path.starts_with ("https") then
+					api_get_call (a_path, Void)
+				else
+					api_get_call (facebook_url (a_path, Void ), Void)
+				end
+			end
+			if
+				attached last_response as l_response and then
+				attached l_response.body as l_body
+			then
+				Result := l_body
+			end
+		end
+
+	user_groups (a_path: STRING; a_params: detachable FB_GROUP_PARAMETER): detachable STRING
+		do
+			if
+				attached a_params and then
+				attached a_params.parameters as l_parameters
+			then
+				api_get_call (facebook_url (a_path, l_parameters ), Void)
+			else
+				if a_path.starts_with ("https") then
+					api_get_call (a_path, Void)
+				else
+					api_get_call (facebook_url (a_path, Void ), Void)
+				end
+			end
+			if
+				attached last_response as l_response and then
+				attached l_response.body as l_body
+			then
+				Result := l_body
+			end
+		end
+
 feature -- Feeds: publish, delete, update
 
-	user_feed_publish (a_user_id: STRING; a_params: detachable FB_USER_FEED_PUBLISHING): detachable STRING
+	publish_on_user_feed (a_user_id: STRING; a_params: detachable FB_USER_FEED_PUBLISHING): detachable STRING
```

Comment:
missing comment 
What is the result for this function that has a procedure name?

Reply:
We need to review the API since it was based on v2.9 of FB API and it seems a lot of things are deprecated now. The comments are not there but, they are part of the class FACEBOOK_I.

> What is the result for this function that has a procedure name?

Yes it looks weird, but commands in FB API returns results :) , in this case it returns the ID of the 
published feed or an error, so our FACEBOOK_I need to be enhaced to not follow the FB API